am-utils-6.2 - update configure.ac

From: Ian Kent <raven@themaw.net>

Make some changes to configure.ac suggested by autoupdate from autoconf
2.71.

Also move AC_USE_SYSTEM_EXTENSIONS and AC_SEARCH_LIBS([strerror],[cposix])
further up to elliminate some warnings.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 configure.ac |   18 +++++++-----------
 1 file changed, 7 insertions(+), 11 deletions(-)

diff --git a/configure.ac b/configure.ac
index e1bc3290..ef4dab0e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,9 +12,7 @@ dnl ######################################################################
 dnl INITIALIZATION
 dnl AC_INIT
 dnl new init style: PACKAGE, VERSION, BUG-REPORTING-ADDRESS
-AC_INIT([am-utils],
-	sinclude([vers.m4]),
-	[https://bugzilla.am-utils.org/ or am-utils@am-utils.org])
+AC_INIT([am-utils],sinclude([vers.m4]),[https://bugzilla.am-utils.org/ or am-utils@am-utils.org])
 AC_MSG_NOTICE(*** INITIALIZATION ***)
 AC_CONFIG_SRCDIR([amd/amd.c])
 AC_CONFIG_MACRO_DIR([m4])
@@ -55,7 +53,7 @@ AH_BOTTOM([
 ])
 dnl
 dnl AC_CONFIG_AUX_DIR(m4)
-AC_PREREQ(2.52)
+AC_PREREQ(2.69)
 AC_REVISION($Revision: 1.134 $)
 AC_COPYRIGHT([Copyright (c) 1997-2011 Erez Zadok])
 dnl find out system type
@@ -105,14 +103,14 @@ dnl ======================================================================
 dnl Particular Program Checks
 AC_MSG_NOTICE(*** PARTICULAR PROGRAMS (part 1) ***)
 dnl AC-PROG-LIBTOOL will run this, but we need this to squelch warnings
-dnl from autoconf about AMU_TRY_COMPILE running before AC_AIX/AC_ISC_POSIX
+dnl from autoconf about AMU_TRY_COMPILE running before AC_USE_SYSTEM_EXTENSIONS/AC_SEARCH_LIBS([strerror],[cposix])
+AC_USE_SYSTEM_EXTENSIONS
+AC_SEARCH_LIBS([strerror],[cposix])
 AC_PROG_CC
 AC_PROG_CPP
 AC_PROG_CC_C_O
 AC_PROG_GCC_TRADITIONAL
 AC_PROG_MAKE_SET
-AC_ISC_POSIX
-AC_AIX
 dnl XXX: available in autoconf-2.14
 dnl AC_PROG_CC_STDC
 dnl ======================================================================
@@ -148,7 +146,7 @@ dnl my version of prog_libtool will not build shared by default
 dnl AM_PROG_LIBTOOL2
 dnl build static libraries by default
 AC_DISABLE_SHARED
-dnl XXX: for autoconf-2.14: use AC_PROG_LIBTOOL, not AM_PROG_LIBTOOL
+dnl XXX: for autoconf-2.69: use LT_INIT, not AC_PROG_LIBTOOL
 case ${host_os} in
 	# Temporarily undo renaming of "linux-gnu" to "linux", because
 	# libtool only looks for "linux-gnu".  Sigh.
@@ -156,7 +154,7 @@ case ${host_os} in
 esac
 dnl this line must appear only once and on its own line
 dnl (hence the double case statements above and below)
-AC_PROG_LIBTOOL
+LT_INIT
 case ${host_os} in
 	# end temporary renaming of "linux-gnu" to "linux"
 	linux-gnu ) host_os=linux ;;
@@ -175,8 +173,6 @@ dnl ######################################################################
 dnl Some of these macros must be called before any call to the C compiler
 AC_MSG_NOTICE(*** UNIX VARIANTS ***)
 dnl UNIX VARIANTS
-dnl AC_AIX
-dnl AC_ISC_POSIX
 AMU_LINUX_HEADERS
 dnl ======================================================================
 
